$NetBSD: patch-configure,v 1.6 2017/01/16 13:13:13 fhajny Exp $

Cripple iODBC check, we're using UnixODBC and it was getting in the way.
Fix Python 3 lookup.
Fix Ruby support.

--- configure.orig	2016-12-20 08:40:38.000000000 +0000
+++ configure
@@ -26775,7 +26775,7 @@ fi
 
 
 
-SEARCHPATH=$ODBCPATH
+SEARCHPATH=OFF_$ODBCPATH
 NAME=iodbc
 HEADER=sql.h
 LIBNAME=iodbc
@@ -32426,12 +32426,12 @@ then
 
 			for i in "3.9" "3.8" "3.7" "3.6" "3.5" "3.4" "3.3" "3.2" "3.1" "3.0"
 			do
-				if ( test -d "$PYTHON3PATH/include/python$i" -a -d "$PYTHON3PATH/lib64/python$i/config" )
+				if ( test -d "$PYTHON3PATH/include/python$i" -a -d "$PYTHON3PATH/lib64/python$i/config-$i" )
 				then
 					PYTHON3INCLUDES="-I$PYTHON3PATH/include/python$i"
 					PYTHON3DIR="$PYTHON3PATH/lib64/python$i"
 				else
-					if ( test -d "$PYTHON3PATH/include/python$i" -a -d "$PYTHON3PATH/lib/python$i/config" )
+					if ( test -d "$PYTHON3PATH/include/python$i" -a -d "$PYTHON3PATH/lib/python$i/config-$i" )
 					then
 						PYTHON3INCLUDES="-I$PYTHON3PATH/include/python$i"
 						PYTHON3DIR="$PYTHON3PATH/lib/python$i"
@@ -32938,7 +32938,7 @@ print CONFIG["arch"]
 print "\n"
 print "ruby_version = "
 begin
-print Config::CONFIG["ruby_version"]
+print RbConfig::CONFIG["ruby_version"]
 rescue
 print CONFIG["ruby_version"]
 end
